configs/AM62AX/AM62AX_linux_toc.txt

```diff
@@ -14,7 +14,6 @@ devices/AM62AX/linux/Release_Specific_Kernel_Performance_Guide
 devices/AM62AX/linux/Linux_Performance_Guide
 devices/AM62AX/linux/RT_Linux_Performance_Guide
 devices/AM62AX/linux/Release_Specific_Supported_Platforms_and_Versions
-devices/AM62AX/linux/Release_Specific_QSG
```

Comment:
@cshilwant , any specific reason for removing this, I see it is present under this directory with name _Release_Specific_QSG.rst

Reply:
https://github.com/TexasInstruments/processor-sdk-doc/pull/132#discussion_r1889001624